src/rmp/src/Restructure.cpp

```diff
@@ -323,7 +337,7 @@ void Restructure::getEndPoints(sta::PinSet& ends,
       if (expanded.size() / 2 > max_depth) {
         ends.insert(end_point->pin());
         // Use only one end point to limit blob size for timing
-        break;
+        // break;
```

Comment:
This may produce a huge blob for while abc will take a long time.  It would be better to iteratively fix cones.